index.Rmd

```diff
@@ -32,32 +32,51 @@ After completing these modifications, your `.Rmd` should look like this [sample
 
 2.  Completely delete the YAML header (the section at the top of the `.Rmd` that includes name, title, date, output, etc.) including the `---` line.
     
-3. Choose a short, descriptive, human readable title for your project as your title will show up in the table of contents -- look at examples in the [rendered book](https://jtr13.github.io/cc20/){target="_blank"}. Capitalize the first letter only ("sentence case"). On the first line of your document, enter a single hashtag, followed by a single whitespace, and then your title. It is important to follow this format so that bookdown renders your title as a header. Do not use single # headers anywhere else in the document.
+3. Choose a short, descriptive, human readable title for your project as your title will show up in the table of contents -- look at examples in the [rendered book](https://jtr13.github.io/cc20/){target="_blank"}. Capitalize the first letter only ("sentence case"). On the first line of your document, enter a single hashtag, followed by a single whitespace, and then your title. It is important to follow this format so that bookdown renders your title as a header. Do not use single `#` headers anywhere else in the document. Note: if your chapter is the first in a section, a second `#` header and section description will be requested during the pull-request and merge phase.
```

Comment:
Actually at least for now, let's link to last year's project (and state so explicitly) since there is only one example in the current rendered book.

Reply:
This was in the index before my changes, but I can update it to link to last year's project. I'll keep the reference in step `1.` to this year's project.